internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -368,9 +368,21 @@ func buildInitialCluster(ctx context.Context, r runtime.Runtime, name, ip string
 		)
 
 		if time.Since(lastNag) > 30*time.Second {
-			log.Printf("etcd is waiting to join the cluster, if this node is the first node in the cluster, please run `talosctl bootstrap`")
-
 			lastNag = time.Now()
+
+			log.Printf("etcd is waiting to join the cluster, if this node is the first node in the cluster, please run `talosctl bootstrap` against one of the following IPs:")
```

Comment:
I like the way it is now, if we want more compact output, we can do `log.Printf("etcd ... following IPs: %s", ips)`

Reply:
this should print something like `[127.0.0.1 10.0.0.5]`

Reply:
yeah, I liked the grouping better. updated a bit:

![Screenshot from 2021-12-23 09-11-33](https://user-images.githubusercontent.com/1529818/147252119-d8ac6ba0-b84e-4971-b21e-731dca327c3a.png)